src/web/components/TeamMember/TeamMembersTable.tsx

```diff
@@ -28,15 +30,33 @@ function TeamMembersTableContent({
   onRemoveTeamMember,
   onUpdateTeamMember,
 }: TeamMembersTableProps) {
+  const { LoggedInUser } = useContext(CurrentUserContext);
+  const { participant } = useContext(ParticipantContext);
+  const userRolesForCurrentParticipant = LoggedInUser?.user?.participants?.find(
+    (p) => p.id === participant?.id
+  )?.currentUserRoleIds;
+
   const [showTeamMemberDialog, setShowTeamMemberDialog] = useState<boolean>(false);
+  const [showTeamMemberActions, setShowTeamMemberActions] = useState<boolean>(false);
 
   const onOpenChangeTeamMemberDialog = () => {
     setShowTeamMemberDialog(!showTeamMemberDialog);
   };
 
-  const { sortData } = useSortable<UserDTO>();
+  const { sortData } = useSortable<UserResponse>();
   const sortedTeamMembers = sortData(teamMembers);
 
+  useEffect(() => {
```

Comment:
I don't think we actually need this `useEffect` or the corresponding `useState`. We can just use a local variable:
```
  const isUserAdminOrSupport =
    LoggedInUser?.user?.isUid2Support ||
    [UserRoleId.UID2Support, UserRoleId.Admin].some((role) =>
      userRolesForCurrentParticipant?.includes(role)
    );
```

Then change all the conditional rendering to use `isUserAdminOrSupport` directly.

Reply:
Changed!